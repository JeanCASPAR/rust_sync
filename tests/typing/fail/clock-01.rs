use rust_sync::sync;

sync! {
    #![pass(1)]
    
    node oui() = ()
    where
        a : float = 3.0 when c;
}

fn main() {}
