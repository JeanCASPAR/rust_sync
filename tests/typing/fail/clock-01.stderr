error: Variable `c` is undefined.
 --> tests/typing/fail/clock-01.rs:8:30
  |
8 |         a : float = 3.0 when c;
  |                              ^
